tools/cdb2sql/cdb2sql.cpp

```diff
@@ -297,50 +321,72 @@ char *generic_generator(const char *text, int state)
     char sql[256];
     //TODO: escape text
     snprintf(sql, sizeof(sql), 
-            "SELECT DISTINCT candidate "
-            "FROM comdb2_completion('%s')", text);
+             "SELECT DISTINCT candidate FROM comdb2_completion('%s')", text);
 
     return db_generator(state, sql);
 }
 
 
 
 // Custom completion function
+//
+// text is the last word entered or space, 
+// start and end are the offsets of that last word in the rl_line_buffer
+// So if we are trying to complete in the middle of a word
+// text is the [partial] word, example: 
+//  > abra kadabr
+//   rl_line_buffer='abra kadabr' text='kadabr' start=5 end=11
```

Comment:
I'd refrain from using such words, may be just use examples pertaining to comdb2/cdb2sql?

Reply:
ok : )